.github/CHANGELOG.md

```diff
@@ -15,6 +15,17 @@
 * Implements caching for Kokkos installation.
 [(#316)](https://github.com/PennyLaneAI/pennylane-lightning/pull/316)
 
+* Supports measurements of operator arithmetic classes such as `Sum`, `Prod`,
+  and `SProd` by deferring handling of them to `DefaultQubit`.
+  [(#349)](https://github.com/PennyLaneAI/pennylane-lightning/pull/349)
+
+```
+@qml.qnode(qml.device('lightning.qubit', wires=2))
+def circuit():
+    obs = qml.s_prod(2.1, qml.PauliZ(0)) + qml.op_sum(qml.PauliX(0), qml.PauliZ(1))
```

Comment:
Given the structure of these, we could obtain a sizeable speed-up by lowering them to the appropriate C++ calls we already have in place. Though, I think that can remain a v0.27 priority.

Reply:
Definitely, though doing so will be outside my C++ abilities.

Reply:
@mlxd Yeap. I will write a simple C++ code (in fact, it already exists) constructing a sparse matrix from a sum of products of Pauli operators and want to compare its performance with an existing Python code in PennyLane.